erpnext/controllers/stock_controller.py

```diff
@@ -558,8 +558,8 @@ def future_sle_exists(args):
 	or_conditions = []
 	for warehouse, items in warehouse_items_map.items():
 		or_conditions.append(
-			"warehouse = '{}' and item_code in ({})".format(
-				warehouse,
+			"warehouse = {} and item_code in ({})".format(
+				frappe.db.escape(warehouse),
 				", ".join(frappe.db.escape(item) for item in items)
 			)
```

Comment:
```suggestion
			f"""warehouse = {frappe.db.escape(warehouse)}
				and item_code in ({', '.join(frappe.db.escape(item) for item in items)})"""
```


F-strings would be better for readability here 🤔